packages/mdc-drawer/README.md

```diff
@@ -437,55 +437,92 @@ document.body.addEventListener('MDCDrawer:closed', () => {
-`removeClass(className: string) => void` | Removes a class from the root element.
-`elementHasClass(element: !Element, className: string) => boolean` | Returns true if the an element contains the given class.
-`saveFocus() => void` | Saves the focus of currently active element.
-`restoreFocus() => void` | Restores focus to element previously saved with 'saveFocus'.
-`focusActiveNavigationItem() => void` | Focuses the active / selected navigation item.
-`notifyClose() => void` | Emits the `MDCDrawer:closed` event.
-`notifyOpen() => void` | Emits the `MDCDrawer:opened` event.
-`trapFocus() => void` | Traps focus on root element and focuses the active navigation element.
-`releaseFocus() => void` | Releases focus trap from root element which was set by `trapFocus` and restores focus to where it was prior to calling `trapFocus`.
-
-### Foundations
-
-#### `MDCDismissibleDrawerFoundation`
-
-Method Signature | Description
---- | ---
-`open() => void` | Opens the drawer from the closed state.
-`close() => void` | Closes the drawer from the open state.
-`isOpen() => boolean` | Returns true if the drawer is in the open position.
-`isOpening() => boolean` | Returns true if the drawer is animating open.
-`isClosing() => boolean` | Returns true if the drawer is animating closed.
-`handleKeydown(evt: Event) => void` | Handles the `keydown` event.
-`handleTransitionEnd(evt: Event) => void` | Handles the `transitionend` event when the drawer finishes opening/closing.
 
-#### `MDCModalDrawerFoundation` (extends `MDCDismissibleDrawerFoundation`)
+### MDCDrawerAdapter
 
-Method Signature | Description
---- | ---
-`handleScrimClick() => void` | Handles click event on scrim.
+#### Methods
+
+Name | Signature | Description
+--- | --- | --- 
+notifyOpen | `() => void` | Emits a custom event "MDCDrawer:opened" denoting the drawer has opened.
+addClass | `(className: string) => void` | Adds a class to the root Element.
+focusActiveNavigationItem | `() => void` | Focuses the active / selected navigation item.
+hasClass | `(className: string) => boolean` | Returns true if the root Element contains the given class.
+notifyClose | `() => void` | Emits a custom event "MDCDrawer:closed" denoting the drawer has closed.
+elementHasClass | `(element: Element, className: string) => boolean` | Returns true if the an element contains the given class.
+releaseFocus | `() => void` | Releases focus trap from root element which was set by `trapFocus` and restores focus to where it was prior to calling `trapFocus`.
+removeClass | `(className: string) => void` | Removes a class from the root Element.
+restoreFocus | `() => void` | Restores focus to element previously saved with 'saveFocus'.
+saveFocus | `() => void` | Saves the focus of currently active element.
+trapFocus | `() => void` | Traps focus on root element and focuses the active navigation element.
+
+
+### MDCModalDrawerFoundation
+
+#### Methods
+
+Name | Signature | Description
+--- | --- | --- 
+init | `() => void` | n/a
```

Comment:
Can we not include it in table if TS doc is missing?

Reply:
removed